packages/hooks/src/useP2PAdvertList.ts

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useFetch, useInvalidateQuery } from '@deriv/api';
+import useExchangeRate from './useExchangeRate';
+
+/**
+ * This custom hook returns available adverts for use with 'p2p_order_create' by calling 'p2p_advert_list' endpoint
+ */
+const useP2PAdvertList = () => {
+    const { getRate } = useExchangeRate();
+    const invalidate = useInvalidateQuery();
+    const { data, ...rest } = useFetch('p2p_advert_list');
+
+    // Add additional information to the 'p2p_advert_list' data!
```

Comment:
```suggestion
    // Add additional information to the 'p2p_advert_list' data.
```
@niloofar-deriv Why always `!`? 😅

Reply:
I like thissss :joy: will remove that :grin: